src/components/Icon/assets/play-count.svg

```diff
@@ -0,0 +1,11 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<svg width="20px" height="20px" viewBox="0 0 20 20" version="1.1" xmlns="http://www.w3.org/2000/svg" xmlns:xlink="http://www.w3.org/1999/xlink">
+    <title>7D28CFE1-96A2-40FB-B4B4-6D89377C1FB4@1x</title>
```

Comment:
Same here. Remove title because it will add tooltip to the icon

Reply:
Nothing changed on OSX but i trust you 😄